compiler/rustc_hir_typeck/src/lib.rs

```diff
@@ -245,12 +247,13 @@ fn typeck_with_fallback<'tcx>(
 
             // Compute the function signature from point of view of inside the fn.
             let fn_sig = tcx.liberate_late_bound_regions(def_id.to_def_id(), fn_sig);
-            let fn_sig = inh.normalize_associated_types_in(
-                body.value.span,
-                body_id.hir_id,
-                param_env,
-                fn_sig,
-            );
+            // FIXME(compiler-errors): Remove
```

Comment:
Not sure I understand this FIXME

Reply:
Should have been fixed in the next commit